mappings/net/minecraft/client/gui/screen/ScreenTexts.mapping

```diff
@@ -7,5 +7,8 @@ CLASS net/minecraft/class_5244 net/minecraft/client/gui/screen/ScreenTexts
 	FIELD field_24337 NO Lnet/minecraft/class_2561;
 	FIELD field_24338 PROCEED Lnet/minecraft/class_2561;
 	FIELD field_24339 BACK Lnet/minecraft/class_2561;
+	FIELD field_26625 FAILED Lnet/minecraft/class_2561;
 	METHOD method_27654 getToggleText (Z)Lnet/minecraft/class_2561;
 		ARG 0 value
+	METHOD method_30619 getComposedToggleText (Lnet/minecraft/class_2561;Z)Lnet/minecraft/class_5250;
```

Comment:
Hmm, I suggest `composeToggleText`
```suggestion
	METHOD method_30619 composedToggleText (Lnet/minecraft/class_2561;Z)Lnet/minecraft/class_5250;
```
Feel free to debate if this is more like a `get` action than a `compose` (i.e. making something new generally) action.